### ISSUE TITLE
Stop using undocumented `CONSTRUCTOR` list

### ISSUE THREAD

**Issue by fingolfin** [gap-packages/Digraphs#109]
Loading digraphs in master now gives a warning, as it accesses the the undocumented internal list `CONSTRUCTOR`, which recently was removed.

The easiest fix is to just remove that check and the associated code:
```
    elif op in CONSTRUCTORS then
      class := "Constructor";
      # seem to never get one
```
as you don't use `DeclareConstructor` in digraphs, it doesn't matter.

Alternatively, you could use `TypeOfOperation`, but that is only available in GAP 4.9 and later.

**Comment by wilfwilson** [comment]
Thanks @fingolfin

**Comment by wilfwilson** [comment]
This is sorted in the newly-released Digraphs v0.12.0.
